projects/plugins/jetpack/_inc/lib/tonesque.php

```diff
@@ -66,6 +68,8 @@ public function __construct( $image_url ) {
 		 * @return object|bool Image object or false if the image could not be loaded.
 		 */
 		public static function imagecreatefromurl( $image_url ) {
+			_deprecated_function( 'Tonesque::imagecreatefromurl', 'jetpack-$$next-version$$' );
```

Comment:
```suggestion
			_deprecated_function( __METHOD__, 'jetpack-$$next-version$$' );
```
Like with https://github.com/Automattic/jetpack/pull/38338/files#r1720052504